pnpm-workspace.yaml

```diff
@@ -0,0 +1,11 @@
+ignoredBuiltDependencies:
+  - tree-sitter-gritql
+
+onlyBuiltDependencies:
+  - '@vscode/vsce-sign'
+  - esbuild
+  - keytar
+  - lefthook
+  - tree-sitter
+  - tree-sitter-javascript
+  - tree-sitter-gritql
```

Comment:
We added all these libraries, but package.json didn't add any of them. Is it a mistake?